lib/pacman/aur-build--sync

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+# shellcheck disable=SC2086
+# build--sync - helper for upgrading local repository
+set -o errexit
+[[ -v AUR_DEBUG ]] && set -o xtrace
+argv0=build--sync
+PS4='+(${BASH_SOURCE}:${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
+
+# default options
+conf_args=() sync_args=() db_names=()
+
+args_csv() {
+    # shellcheck disable=SC2155
+    local str=$(printf '%s,' "$@")
+    printf '%s' "${str%,}"
+}
+
+get_local_config() {
+    #global conf_args
+    local repo
+
+    printf '[options]\n'
+    pacconf "${conf_args[@]}" --options --raw
+
+    for repo in "$@"; do
```

Comment:
```sh
for repo do
    printf '[%s]\n' "$repo"
    pacconf "${conf_args[@]}" --repo="$repo" --raw
done
```

Reply:
This could be done in a few other places as well, but not everyone might know about the implied "$@"